src/styles/theme.js

```diff
@@ -0,0 +1,86 @@
+const baseSpacerUnit = 16;
+
+// Breakpoints in REM
+export const breakpoints = {
+  xs: '0rem',
+  sm: '32rem',
+  md: '48rem',
+  lg: '64rem',
+  xl: '75rem',
+};
+
+// Breakpoints in pixels using above breakpoints
+export const screenSizes = {
+  small: Number(breakpoints.sm.slice(0, -3)) * 16,
```

Comment:
```suggestion
  small: parseFloat(breakpoints.sm) * 16,
```

Super nit-picky here, but a little more terse and resilient.

Reply:
nice 👍